src/Organizer/WebsiteNormalizer.php

```diff
@@ -4,24 +4,23 @@
 
 namespace CultuurNet\UDB3\Organizer;
 
-use ValueObjects\Web\PortNumber;
-use ValueObjects\Web\Url;
+use CultuurNet\UDB3\Model\ValueObject\Web\Url;
 
 final class WebsiteNormalizer
 {
     public function normalizeUrl(Url $url): string
     {
-        $domain = $url->getDomain()->toNative();
+        $domain = $url->getDomain();
         if (strpos($domain, 'www.') === 0) {
             $domain = substr($domain, strlen('www.'));
         }
 
-        $port = $url->getPort() instanceof PortNumber ? ':' . $url->getPort()->toNative() : '';
+        $port = $url->getPort() ? ':' . $url->getPort() : '';
 
-        $queryString = $url->getQueryString()->toNative();
-        $fragment = $url->getFragmentIdentifier()->toNative();
+        $queryString = $url->getQueryString() ? '?' . $url->getQueryString() : '';
```

Comment:
It seems that the old `Url` implementation did always add `?` to the query and `#` to the fragment. See this snippet inside the old `Url` implementation:
```
        $query      = $queryString ? new QueryString(\sprintf('?%s', $queryString)) : new NullQueryString();
        $fragment   = $fragmentId ? new FragmentIdentifier(\sprintf('#%s', $fragmentId)) : new NullFragmentIdentifier();
```

Now the new methods return those fields without the prefix, that matches also the returned values from the built-in PHP method `parse_url`. So I think this new implementation is better, what do you prefer @bertramakers ?

Reply:
Without sounds good!

Reply:
I don't consider the `?` part of the query parameter(s) or `#` part of the fragment. They just indicate where the query parameter(s) and/or fragment start.